packages/block-serialization-default-parser/README.md

```diff
@@ -0,0 +1,67 @@
+# Block Serialization Default Parser
+
+This library contains the default block serialization parser implementations for WordPress documents. It provides native PHP and JavaScript parsers that implement the specification from `@wordpress/block-serialization-spec-parser` and which normally operates on the document stored in `post_content`.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/block-serialization-default-parser --save
+```
+
+## Usage
+
+```js
+import { parse } from '@wordpress/block-serialization-default-parser';
+
+parse( '<!-- wp:core/more --><!--more--><!-- /wp:core/more -->' ) === [
+    {
+        "attrs": null,
+        "blockName": "core/more",
+        "innerBlocks": [],
+        "innerHTML": "<!--more-->"
+    }
+]
+```
```

Comment:
I suggest a more readable example. `core/more` is pretty quirky, since it's wrapping a core magical string `<!--more-->`. Anything else, really, as long as it's short and sweet.

Reply:
here I was just copying the same example from the spec parser which was pretty minimal but in 66455b461 I made a bigger example